Feature/m0078 ddg 297 document get sources (#318)
MicroEJ/docs: ApplicationDeveloperGuide/launches.rst

-  To set thread stack space to 512MB, set the ``-Xss512M`` option.

.. _launch_source_tab:

Source Tab
----------

The next tab is the :guilabel:`Source` tab.
By default, it is automatically configured to connect your Add-On Libraries sources dependencies.
To connect your Platform Foundation Library sources, please refer to the section :ref:`foundation_library_sources`.

Common Tab
----------

> The last tabs is the :guilabel:`Common` tab. This is a default Eclipse tab that allows to configure you launch 

Reviewer: ```suggestion
The last tab is the :guilabel:`Common` tab. This is a default Eclipse tab that allows to configure your launch.
```
